### DISCUSSION TITLE
trending

### DISCUSSION THREAD

**Discussion by anclopezg** [davidteather/TikTok-Api#713] [General]
Hello everyone, first I want to thank David what a genius, second I would like to ask about the api.trending (), with all the other functions I managed to get less results with this one. I appreciate if someone can help me the error is "AttributeError: 'TikTokApi' object has no attribute 'trending'"

**Comment by davidteather** [comment]
Things got switched to camel case in [V4.0.0](https://github.com/davidteather/TikTok-Api/releases/tag/V4.0.0) and `trending` got replaced with `by_trending` you should just be able to switch those out :)

**Reply by anclopezg** [reply]
thank's bro
